babel.config.cjs

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  presets: [
+    '@babel/preset-env', // Transforms modern JavaScript
+    '@babel/preset-typescript', // Transforms TypeScript
+    '@babel/preset-react', // Transforms JSX
```

Comment:
1. Can this file be placed in a `.config` or `config` directory like we have done with the API?
2. It'll reduce the clutter in the root directory